src/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_css.md

```diff
@@ -117,7 +118,7 @@ For grouping style rules in certain media queries the `.media-width()` mixin use
 }
 ```
 
-You can find more information about the Magento UI library responsive mixin usage in `<your_Magento_instance>/pub/static/frontend/Magento/blank/en_US/css/docs/responsive.html` (view in a browser).
+You can find more information about the Magento UI library responsive mixin usage in `<your_Magento_instance>/lib/web/css/docs/responsive.html` (view in a browser).
```

Comment:
Hello @mrtuvn.
I think the previous example was correct, and it properly works in clean Magento with default localization settings.
Also note, the content of internal `lib` folder should not be exposed to the frontend on the properly configured system, so it won't be accessible in the browser.

Reply:
ooh i see. But previous we can't access it so i think we should remove (view in a browser)

Reply:
I still don't see the reasons to remove the proper example of an autogenerated page `<your_Magento_instance>/pub/static/frontend/Magento/blank/en_US/css/docs/responsive.html`.
I've checked it on the latest `2.4-develop` branch, and it works as expected.

![Screenshot 2020-05-22 at 14 20 46](https://user-images.githubusercontent.com/11693779/82663111-bca0a380-9c37-11ea-8da4-2b8a7505d312.png)

Reply:
@dmytro-ch how to generate ? I don't see this file in my path magento instance
No such folder named docs lay down in pub/static/frontend/Magento/blank/en_US/css
404 Not Found
nginx/1.16.1

Reply:
I still don't see any reason to remove the proper example of an autogenerated page <your_Magento_instance>/pub/static/frontend/Magento/blank/en_US/css/docs/responsive.html.

Magento not autogenerated that file in pub

Reply:
No, it's automatically generated when the page is requested (on demand) on clean Magento installation in `developer` mode.
```
bin/magento dep:mode:set developer; rm -rf pub/static/*; rm -rf var/view_preprocessed/*;
```